### ISSUE TITLE
Opening a new project comment warnings 

### ISSUE THREAD

**Issue by benloh** [theRAPTLab/meme-2023#5]
1. Open a project
2. Click the Comment button to the right of the project name
3. Multiple warnings:

```
Warning: Failed prop type: Invalid prop `className` of type `object` supplied to `ForwardRef(Typography)`, expected `string`.
    at Typography (
```

```
Warning: Failed prop type: Invalid prop `className` of type `object` supplied to `ForwardRef(InputLabel)`, expected `string`.
    at InputLabel (
```

etc. 7 warnings in all

---

Copied from https://github.com/Vertex-Labs/meme/issues/7

**Comment by benloh** [comment]
This commit is an example of a successful fix for this kind of error: 42d38f5620660143174b24f9fba714fe93db5f18

Basically it requires the use of the `styled` utility.

**Comment by benloh** [comment]
Should be fixed with #25.
